infrastructure/src/main/java/org/corfudb/infrastructure/ManagementServer.java

```diff
@@ -174,11 +186,11 @@ private CorfuRuntime getNewCorfuRuntime() {
     }
 
     /**
-     * Handler for this server.
+     * HandlerMethod for this server.
      */
     @Getter
-    private final CorfuMsgHandler handler =
-            CorfuMsgHandler.generateHandler(MethodHandles.lookup(), this);
+    private final HandlerMethods handler =
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/app/CorfuDB/CorfuDB/pullRequest?prid=4662281)